compiler/rustc_infer/src/infer/outlives/test_type_match.rs

```diff
@@ -13,9 +13,11 @@ use crate::infer::region_constraints::VerifyIfEq;
 
 /// Given a "verify-if-eq" type test like:
 ///
+/// ```ignore (not valid syntax)
```

Comment:
It looks like it would be better to align the description with other parts of the document and make it ignore(illustrative).